packages/myst-to-tex/src/index.ts

```diff
@@ -80,10 +80,18 @@ const handlers: Record<string, Handler> = {
   definitionDescription(node, state) {
     state.renderChildren(node, true);
   },
-  code(node, state) {
-    state.write('\\begin{verbatim}\n');
+  code(node: Code, state) {
+    let start = '\\begin{verbatim}\n';
+    let end = '\n\\end{verbatim}'
+
+    if (node.class === 'minted' && node.lang !== undefined) {
```

Comment:
I think the only improvements I would suggest to this is that the class might be a space-separated (i.e. css) string.

Something similar to:

```typescript
function getClasses(className?: string): string[] {
  const classes =
    className
      ?.split(' ')
      .map((s) => s.trim().toLowerCase())
      .filter((s) => !!s) ?? [];
  return [...new Set(classes)];
}
```

and then a `getClasses(node.class).includes('minted')` should work well for this use case I think.